doc/source/tutorial/stats/continuous_kstwo.rst

```diff
@@ -0,0 +1,114 @@
+
+.. _continuous-kstwo:
+
+KStwo Distribution
+==================
+
+This is the distribution of the maximum absolute differences between an
```

Comment:
Let's add that the underlying distribution of the samples is assumed to be continuous

Reply:
Agreed.